apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/style.scss

```diff
@@ -22,11 +22,11 @@
 
 .post-type-page, .post-type-wp_template_part {
 	@media ( min-width: 768px ) {
-		.edit-post-layout__content {
+		.edit-post-editor-regions__content {
```

Comment:
Are y'all going to wait for the update to land this? Otherwise adding additive rules might help here:

```
.edit-post-layout__content, 
.edit-post-editor-regions__content {
//...
.edit-post-layout__content .edit-post-visual-editor,
.edit-post-editor-regions__content .edit-post-visual-editor {
```